src/styles/body.scss

```diff
@@ -0,0 +1,9 @@
+.body {
```

Comment:
The `.body` class is used instead of the `body` element selector. If you intend to style the HTML body element, please correct this to ensure styles are applied as intended.